delocate/cmd/delocate_fuse.py

```diff
@@ -1,15 +1,14 @@
 #!/usr/bin/env python3
 """Fuse two (probably delocated) wheels.
 
-Overwrites the first wheel in-place by default.
+Writes to a new wheel with an automatically determined name by default.
 """
 
 # vim: ft=python
 from __future__ import absolute_import, division, print_function
```

Comment:
Update `__future__` for Python 3 code. Fixes Mypy errors.
```suggestion
from __future__ import annotations
```

Reply:
I marked this on the wrong file. `from __future__ import annotations` needs to be added to `delocate/fuse.py`.

Reply:
I'll add it to both.